packages/product-react/src/components/List/List.tsx

```diff
@@ -0,0 +1,58 @@
+import clsx from 'clsx';
+import React from 'react';
+import { Avatar } from '../Avatar';
+import './List.scss';
+
+export interface IItems {
+  header?: React.ReactNode;
+  content?: React.ReactNode;
+  icon?: React.ReactNode;
+}
+
+export interface IListProps extends React.HTMLProps<HTMLLIElement> {
+  items: IItems[];
+  listItemComponent?: any;
+  onClick?(event: React.MouseEvent<HTMLLIElement>): void;
+}
+
+const ListItem: React.FC<IListProps> = ({ items, className, ...rest }) => {
+  let listItems: React.ReactNode[];
+  listItems = items.map((item: IItems, index: number) => {
+    return (
+      <li className={className} {...rest} key={index}>
+        {item.icon && (
+          <Avatar
+            className="icon-block"
+            image={String(item.icon)}
```

Comment:
Will this ever not be a string? Could that instead be enforced via typescript?

Reply:
what about the fallback `name` property for Avatar?

Reply:
uhmm good point let me investigate with @seimithwework

Reply:
Oh yeah, we can set the `image` type to string. What do you by fallback `name` property? Are you talking about when you pass in a "name" to the `Avatar` and we grab the first initial and display that? @Morgantheplant

Reply:
> Will this ever not be a string? Could that instead be enforced via typescript?

yes it's already enforced, inside the avatar image is the type of string, I think the image will always be base64, so if pass a base64, should work also. will remove the string part and will type it of string